features/product/managing_product_reviews/editing_product_review.feature

```diff
@@ -0,0 +1,45 @@
+@managing_product_reviews
+Feature: Editing product reviews
+    In order to correct customer's opinion about my products
+    As an Administrator
+    I want to edit a product review
+
+    Background:
+        Given the store has customer "Mike Ross" with email "ross@teammike.com"
+        And the store has a product "Lamborghini Gallardo Model"
+        And this product has a review titled "Awesome" and rated 4 with a comment "Nice product" added by customer "ross@teammike.com"
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Changing a title of a product review
+        When I want to modify the "Awesome" product review
+        And I change its title to "Bewildering"
```

Comment:
It makes this scenarios making harder to understand at the first sight, at least for me.